smart_open/tests/test_smart_open.py

```diff
@@ -191,6 +191,16 @@ def test_open_with_keywords_explicit_r(self):
             actual = fin.read()
         self.assertEqual(expected, actual)
 
+    def test_open_pathlib_path(self):
+        """If ``pathlib.Path`` is available we should open it."""
+        try:
+            from pathlib import Path
+            fpath = Path(os.path.join(CURR_DIR, 'test_data/cp852.tsv.txt'))
+            with smart_open.smart_open(fpath, encoding='cp852') as fin:
+                fin.read()
+        except ImportError:
```

Comment:
That's a bad practice to silently pass the test, better to add a condition about python version for test https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures